contracts/src/Assets.sol

```diff
@@ -68,6 +68,11 @@ library Assets {
             // If the final destination chain is not AssetHub, then the fee needs to additionally
             // include the cost of executing an XCM on the final destination parachain.
             costs.foreign = $.assetHubReserveTransferFee + destinationChainFee;
+
+            // Destination fee cannot be zero. MultiAssets are not allowed to be zero in xcm v4.
+            if (destinationChainFee < 1 || destinationChainFee > $.destinationMaxTransferFee) {
```

Comment:
prefer be more explicit as `if (destinationChainFee == 0 || destinationChainFee > $.destinationMaxTransferFee)`

Reply:
adddressed in [b5ea5fa](https://github.com/Snowfork/snowbridge/pull/1168/commits/b5ea5fa5ce6d201c22c1490002bacded10be290c)